src/rohit/BankingSystem.java

```diff
@@ -0,0 +1,72 @@
+
+package rohit;
+
+public class BankingSystem {
+	int currentbalance;
+	int debitamount;
+	int creditamount;
+	static int alldebit;
+	static int allcredit;
+	static int allprintbalance;
+
+	void currentbalance(int balance) {
+		currentbalance = balance;
+	}
+
+	void debitamount(int debit) {
+		currentbalance = currentbalance - debit;
+		debitamount++;
+		alldebit++;
+	}
+
+	void creditamount(int credit) {
+		currentbalance = currentbalance + credit;
+		creditamount++;
+		allcredit++;
+	}
+
+	void printbalance() {
+
```

Comment:
There is one requirement related this as well for individual user, that you missed here.
i want individual printTotalBalanace count as well.